lib/v2/screens/app/interactor/viewmodels/app_event.dart

```diff
@@ -48,3 +48,17 @@ class ClearAppPageCommand extends AppEvent {
   @override
   String toString() => 'ClearAppPageCommand';
 }
+
+class HandleInitialDeepLink extends AppEvent {
```

Comment:
may be leave some comments explaning the difference between initial deep link and incoming deep link.